fdbserver/fdbserver.actor.cpp

```diff
@@ -1111,6 +1112,25 @@ struct CLIOptions {
 private:
 	CLIOptions() = default;
 
+	void parseEnvInternal() {
+		for (std::string knob : getEnvironmentKnobOptions()) {
+			auto pos = knob.find_first_of("=");
+			if (pos == std::string::npos) {
+				fprintf(stderr,
+				        "Error: malformed environment knob option: %s%s\n",
+				        ENVIRONMENT_KNOB_OPTION_PREFIX,
+				        knob.c_str());
+				TraceEvent(SevWarnAlways, "Error: malformed environment knob option")
```

Comment:
A trace event type name must adhere to a certain format which disallows spaces and punctuation.

```suggestion
				TraceEvent(SevWarnAlways, "MalformedEnvironmentVariableKnob")
```